hydra/core/override_parser/overrides_visitor.py

```diff
@@ -83,6 +83,11 @@ def visitPrimitive(
         last_idx = ctx.getChildCount()
         # skip first if whitespace
         if self.is_ws(ctx.getChild(0)):
+            if last_idx == 1:
```

Comment:
`key=` is supported as a way to set key to the empty string.
not sure I understand the reason for this exception.

Reply:
`key=` still works (there is no WS in that case). In normal use cases it should not be possible to have a primitive made only of whitespaces (because all those whitespaces should have been "consumed" into another lexer token). Also there was a test about this that was expecting a crash. It's the following parameterization of `test_parse_errors` (from current master):

```python
        pytest.param(
            "primitive",
            " ",
            pytest.raises(
                HydraException, match=re.escape("mismatched input '<EOF>' expecting"),
            ),
            id="error:value:whitespace",
        ),
```

Reply:
roger.